enginetest/queries/alter_table_queries.go

```diff
@@ -17,13 +17,33 @@ package queries
 import (
 	"github.com/dolthub/vitess/go/mysql"
 
-	"github.com/dolthub/go-mysql-server/sql/plan"
-
 	"github.com/dolthub/go-mysql-server/sql"
+	"github.com/dolthub/go-mysql-server/sql/analyzer/analyzererrors"
+	"github.com/dolthub/go-mysql-server/sql/plan"
 	"github.com/dolthub/go-mysql-server/sql/types"
 )
 
 var AlterTableScripts = []ScriptTest{
+	{
+		Name: "multi alter with invalid schemas",
+		SetUpScript: []string{
+			"CREATE TABLE t(a int primary key)",
+		},
+		Assertions: []ScriptTestAssertion{
+			{
+				Query:       "alter table t add column b varchar(max)",
+				ExpectedErr: analyzererrors.ErrInvalidRowLength,
+			},
+			{
+				Query:       "alter table t add column b varchar(16000), add column c varchar(16000)",
```

Comment:
Surprising that 32000 characters is too long given the 64K row size. Add a comment about why this is?>

Reply:
Each char is 4 bytes with default collation. With latin1 you can have 64k varchar. I have separate tests for those limits